app/controllers/VideoUIApp.scala

```diff
@@ -18,13 +19,20 @@ class VideoUIApp @Inject() (val authActions: HMACAuthActions)
       case None => routes.Assets.versioned(jsFileName).toString
     }
 
+    val composerUrl = awsConfig.stage match {
+      case "PROD" => "https://composer.gutools.co.uk"
```

Comment:
Should we have this in config? It seems to be the pattern we're adopting for this type of config